flink-autoscaler/src/main/java/org/apache/flink/autoscaler/ScalingMetricCollector.java

```diff
@@ -143,7 +145,11 @@ public CollectedMetricHistory updateMetrics(
         if (now.isBefore(windowFullTime)) {
             LOG.info("Metric window not full until {}", windowFullTime);
         } else {
-            collectedMetrics.setFullyCollected(true);
+            if (isExcluded) {
+                LOG.info("autoscaling now in excluded period");
```

Comment:
```suggestion
                LOG.info("Autoscaling on halt based on exclusion rule {}",
                    conf.get(AutoScalerOptions. EXCLUDED_PERIODS));
```

I think this could be helpful for debugging.

Reply:
Agreed. Adjustment has been made